src/admin.tsx

```diff
@@ -0,0 +1,106 @@
+/**
+ * This part runs inside Rise Playground.
+ *
+ * You can use anything that is available there. As a good practice,
+ * import from `kitchen-sink` and `kit-react-navigation` only.
+ */
+
+import { ActivityIndicator, FlatList } from '@rise-tools/kit-react-native/server'
+import { goBack, navigate, StackScreen } from '@rise-tools/kit-react-navigation/server'
+import {
+  Button,
+  InputField,
+  RiseForm,
+  SubmitButton,
+  Text,
+  toast,
+  View,
+} from '@rise-tools/kitchen-sink/server'
+import { response } from '@rise-tools/react'
+import { lookup, view } from '@rise-tools/server'
+
+import { edition, updateVideo, video } from './db'
+
+function Admin() {
+  return (
+    <>
+      <StackScreen title="React Universe Console" />
+      <View padding="$2">
+        <Text>Choose edition</Text>
```

Comment:
we might want a `ReadOnly<Array<>>` for editions or even make it dynamic based on the db

Reply:
Yeah, the idea is to make it dynamic from the database, I just haven't gotten around to it, because I still have to import all JSONs into the db! Good thought tho!